boards/artix7_35t.json

```diff
@@ -1,31 +1,32 @@
 {
   "build": {
     "extra_flags": "-D ARTIX7_35T",
-    "f_cpu": "320000000L",
+    "f_cpu": "50000000L",
     "hwids": [
       [
-        "0x0403",
-        "0x6010"
+        "0x002a",
+        "0x15ba"
       ]
     ],
     "ldscript": "link.ld",
     "mabi": "ilp32",
     "march": "rv32imac",
     "mcmodel": "medany",
-    "mcu": "e31"
+    "mcu": "E-Class"
   },
   "debug": {
-    "onboard_tools": [
-      "ftdi"
-    ]
+
+	 "onboard_tools": ["ftdi"],
+         "svd_path":"arty35t.svd"
   },
   "frameworks": [
     "shakti-sdk"
   ],
   "name": "Artix-7 35T Arty FPGA Evaluation Kit",
   "upload": {
-    "maximum_ram_size": 268435456,
-    "maximum_size": 16777216,
+    "maximum_ram_size": 0,
+    "maximum_size": 131072,
+    "speed" : 19200,
```

Comment:
I'm a bit confused about these values.
Where is `speed` value used? I see you specified monitor speed in examples, so why do you specify this value in upload section again?
`maximum_ram_size: 0` How do you represent volatile memory in this case? I see there is `256 MB DDR3L` on board, is it used in any way?
`maximum_size: 131072` Any reason why it's exactly 128K, there is `16 MB Quad-SPI Flash` in the specification for this board.

Reply:
Speed i have excluded as i got confused between monitor speed and the speed. For the RAM confusion, we have configured only 128MB (arty_100t) usable as part of the bitstream. Also for 35t there is a TCM of 128KB , but for this case can i put it as part of the "minimum_ram_size"?

Reply:
Details for memory configuration of both boards 

Arty 100t
RAM- 128MB DDR(128MB is usable out of 256MB )
SPI-Not integrated

Arty 35t
RAM- TCM configured upto 128KB
SPI-Not integrated

Reply:
@valeros

Reply:
Thanks for the clarification, let's specify 128MB as `maximum_ram_size` for `Arty 100t` and 128K as  `maximum_ram_size` for `Arty 35t`.
Also, do you mind creating a clean pull request as this one has a lot of "fix" commits. Please and make it consistent with our formatting.

Reply:
Agreed , once i fix the formatting i will close this and create a new PR. One more thing , what about the maximum_size variable ? as we don't have an SPI integrated now. Put it there or other ?

Reply:
Thanks, `maximum_size: 0` should be fine

Reply:
Cool, Will do . Thanks!